iree/samples/static_library/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+################################################################################
+# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
+# iree/samples/simple_embedding/BUILD                                          #
+#                                                                              #
+# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
+# CMake-only content.                                                          #
+#                                                                              #
+# To disable autogeneration for this file entirely, delete this header.        #
+################################################################################
+
+iree_add_all_subdirs()
+
+iree_c_embed_data(
+  NAME
+    static_library_module
+  SRCS
+    "static_library_module.vmfb"
+  C_FILE_OUTPUT
+    "static_module.c"
+  H_FILE_OUTPUT
+    "static_module.h"
+  IDENTIFIER
+    "iree_samples_static_library_module"
+  FLATTEN
+  PUBLIC
+)
+
+if(NOT ${IREE_TARGET_BACKEND_DYLIB-LLVM-AOT} OR NOT ${IREE_TARGET_BACKEND_VULKAN-SPIRV}
+   OR NOT ${IREE_HAL_DRIVER_DYLIB} OR NOT ${IREE_HAL_DRIVER_VULKAN})
+  return()
+endif()
+
+iree_cc_binary(
+  NAME
+    static_library_demo
+  SRCS
+    "static_library_demo.c"
+  DEPS
+    ::static_library_module
+    iree::base
+    iree::hal
+    iree::hal::local
+    iree::hal::local::loaders::static_library_loader
+    iree::hal::local::task_driver
+    iree::modules::hal
+    iree::task::api
+    iree::vm
+    iree::vm::bytecode_module
+    ${CMAKE_CURRENT_SOURCE_DIR}/static_library_module.o
```

Comment:
this should be binary dir - we don't want to be telling users to generate files into their source path (or having the CI do that)